content/en/docs/contributing/release-process.md

```diff
@@ -27,181 +27,211 @@ following conditions:
 3. You need to have the GitHub `write` permission on the cert-manager project.
    To check that you have the `write` role, [get a personal access
    token](https://github.com/settings/tokens) and run:
-     ```sh
-     GH_USER=maelvls
-     curl -sH "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/jetstack/cert-manager/collaborators/$GH_USER/permission
-     ```
-     If your permission is `write` or `admin`, then you are good to go:
-     ```json
-     {
-      "permission": "write",
-      "user": {...}
-     }
-     ```
 
-    To request the `write` permission on the cert-manager project, [open a
-   PR](https://github.com/jetstack/platform-board/pulls/new) with a link to
-   here.
+    ```sh
+    go install github.com/cli/cli/cmd/gh@latest
+    gh auth login
+    gh api /repos/jetstack/cert-manager/collaborators/$(gh api /user | jq -r .login)/permission | jq .permission
+    ```
+
+    If your permission is `write` or `admin`, then you are good to go. To request
```

Comment:
The token you need doesn't need any permissions, is there a better way to determine token validity?

Reply:
Aaah yeah, I mixed up two steps that require different scopes. I think I mixed things up because I don't use SSH for pushing; instead, I rely on a token + HTTPS.

The two things I am mixing up:

1. for pushing the release-1.* to the repo, it requires to have the `write` or `admin` permission on jetstack/cert-manager
2. for running `cmrel` and not hit the API rate limits, it just requires a token with no scope

The above `gh api` command allows you to check (1).

I'll rephrase

Reply:
Update: done in 786c026db5218fde5195114fee5e498d2de5b713